languagetool-language-modules/ga/src/main/java/org/languagetool/rules/ga/DhaNoBeirtRule.java

```diff
@@ -0,0 +1,97 @@
+/* LanguageTool, a natural language style checker
+ * Copyright (C) 2005 Daniel Naber (http://www.danielnaber.de)
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301
+ * USA
+ */
+package org.languagetool.rules.ga;
+
+import org.languagetool.AnalyzedSentence;
+import org.languagetool.AnalyzedToken;
+import org.languagetool.AnalyzedTokenReadings;
+import org.languagetool.rules.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ResourceBundle;
+
+import static org.languagetool.rules.ga.DhaNoBeirtData.getNumberReplacements;
+import static org.languagetool.rules.ga.DhaNoBeirtData.getDaoine;
+
+public class DhaNoBeirtRule extends Rule {
+  public DhaNoBeirtRule(ResourceBundle messages) {
+    super.setCategory(Categories.MISC.getCategory(messages));
+    setLocQualityIssueType(ITSIssueType.Misspelling);
+    addExamplePair(Example.wrong("Tá <marker>dhá</marker> dheartháireacha agam."),
+      Example.fixed("Tá <marker>beirt</marker> deartháireacha agam."));
```

Comment:
Examples work as tests when abstract rules allow them. However, this test is passing, even though the rule is not marking anything in the compile version (after activation in Irish.java activation).